cljfmt/src/cljfmt/core.cljc

```diff
@@ -334,7 +336,7 @@
       zloc)))
 
 (defn- find-namespace [zloc]
-  (some-> zloc top (z/find z/next ns-form?) z/down z/next z/sexpr))
+  (some-> zloc root z/down (z/find z/right ns-form?) z/down z/next z/sexpr))
```

Comment:
Since `ns-form?` is only true for top-level forms, it's more efficient to search `z/right` across the top level forms, rather than `z/next`, down through all the forms.